README.md

```diff
@@ -16,6 +16,9 @@ Here are a few screenshots showing you what it can look like:
 [![sample screenshot](http://i.imgur.com/A6spiZZt.png)](http://i.imgur.com/A6spiZZ.png)
 [![sample screenshot](http://i.imgur.com/TY5a5r9t.png)](http://i.imgur.com/TY5a5r9.png)
 
+You can polybar configs for these example images (and other configs) [here](https://github.com/jaagr/dots/tree/master/.local/etc/themer/themes): 
```

Comment:
Missed a word ("You can _find_"), also should end with a period at the end.

Thanks for the initiative.

Reply:
Sure! Will fix it, thanks.